yarn.lock

```diff
@@ -5203,10 +5203,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/node@npm:*, @types/node@npm:~12.12.0":
-  version: 12.12.70
-  resolution: "@types/node@npm:12.12.70"
-  checksum: 69c193e0bce8fa914d478f829bd923ee662b220e069a2e00ffbe109f27d0b38fbf2a9fe78c99eb4d4ccb294248ff8de68019a2c7fadb032d301c6feca25f3d22
+"@types/node@npm:*, @types/node@npm:^17.0.5":
```

Comment:
`*` should refer to 12 still (we want that to be the version used in the repo)

Reply:
could we add a resolution entry in package.json to make @types/node always resolve to v12? is this a good idea?

Reply:
now * is resolved to v17, so typescript is failing to compile.